web/beacon-app/src/features/topics/components/__tests__/TopicQuickView.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+/// import { dynamicActivate } from '../../../../I18n';
+import TopicQuickView from '../TopicQuickView';
+
+// mock SentryErrorBoundary
+vi.mock('SentryErrorBoundary', () => ({
+  __esModule: true,
+  default: ({ children }: any) => <div>{children}</div>,
+}));
+
+// mock Suspense
+vi.mock('Suspense', () => ({
+  __esModule: true,
+  default: ({ children }: any) => <div>{children}</div>,
+}));
+// mock useFetchTopicStats
+const renderComponent = (props: { topicID: string }) => {
+  const queryClient = new QueryClient();
+  const wrapper = ({ children }: any) => (
+    <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+  );
+  return render(<TopicQuickView {...props} />, { wrapper });
+};
+
+vi.mock('../../hooks/useFetchTopicStats', () => ({
+  __esModule: true,
+  default: () => ({
+    topicStats: [
+      {
+        name: 'Online Publishers',
+        value: 1,
+      },
+      {
+        name: 'Online Subscribers',
+        value: 2,
+      },
+      {
+        name: 'Avg Events/ Second',
```

Comment:
During office hours, it was stated that this will be the total number of events and not events per second.